--- server.c.orig	Thu Jun 10 19:12:50 2004
+++ server.c	Mon Jun 11 02:52:47 2007
@@ -4846,7 +4846,7 @@
   REQ_LINE("watch-remove", serve_watch_remove, 0),
   REQ_LINE("watchers", serve_watchers, 0),
   REQ_LINE("editors", serve_editors, 0),
-  REQ_LINE("init", serve_init, RQ_ROOTLESS),
+  REQ_LINE("init", serve_noop, RQ_ROOTLESS),
   REQ_LINE("annotate", serve_annotate, 0),
   REQ_LINE("rannotate", serve_rannotate, 0),
   REQ_LINE("noop", serve_noop, RQ_ROOTLESS),
